admin_scripts/ubuntu/delete_libreoffice_shortcuts.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    delete_libre_office_shortcuts.sh
+#%
+#% DESCRIPTION
+#%    Deletes all libreoffice shortcuts for user.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         delete_libre_office_shortcuts (magenta.dk) 0.0.1
+#-    author          Danni Als
+#-    copyright       Copyright 2017, Magenta Aps"
+#-    license         GNU General Public License
+#-    email           danni@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2017/06/10 : danni : Added header
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+CALCDESKTOP="/home/.skjult/Desktop/calc.desktop"
+
+if [ -f $CALCDESKTOP ]
+then
+    rm -f "/home/.skjult/Desktop/calc.desktop"
+else
+    echo "File does not exist"
+    exit -1
+fi
+
+WRITERDESKTOP="/home/.skjult/Desktop/writer.desktop"
+
+if [ -f $WRITERDESKTOP ]
+then
+    rm -f "/home/.skjult/Desktop/writer.desktop"
+else
+    echo "File does not exist"
+    exit -1
+fi
+
+IMPRESSDESKTOP="/home/.skjult/Desktop/impress.desktop"
+
+if [ -f $IMPRESSDESKTOP ]
+then
+    rm -f "/home/.skjult/Desktop/impress.desktop"
+else
+    echo "File does not exist"
+    exit -1
+fi
```

Comment:
`rm -f` ignores fatal errors, which doesn't seem to be what we want here (since every missing file is then explicitly turned into a fatal error). What about just `set -e` at the top followed by three plain `rm` commands?

Reply:
I have made the changes recommended